src/components/layout/Navbar.js

```diff
@@ -10,35 +29,23 @@ function Navbar() {
         </Link>
         <nav className="header__navigation">
           <ul>
-            <li>
-              <NavLink
-                to="/"
-                className={({ isActive }) => (isActive ? activeClassName : undefined)}
-              >
-                Home
-              </NavLink>
-            </li>
-            <li>
-              <NavLink
-                to="/about"
-                className={({ isActive }) => (isActive ? activeClassName : undefined)}
-              >
-                About
-              </NavLink>
-            </li>
-            <li>
-              <NavLink
-                to="/team"
-                className={({ isActive }) => (isActive ? activeClassName : undefined)}
-              >
-                Our Team
-              </NavLink>
-            </li>
+            {navLinks.map((link) => {
+              return (
+                <li key={`link-${link.name}`}>
+                  <NavLink
+                    className={({ isActive }) => (isActive ? activeClassName : undefined)}
```

Comment:
ale to jest tu zbędne moim zdaniem, NavLink automatycznie dodaje class "active", kiedy jest aktywny